attachment_action/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Actions on attachments",
+    "version": "8.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Hidden/Dependency",
+    "summary": "Enable/disable actions on attachments",
+    "depends": [
+        'web',
+        'knowledge',
+        'document',
```

Comment:
Are all these dependencies strictly necessary?

Reply:
thanks for the review Daniel. We indeed only need `web` for technical reasons, but `document` makes sense imho because otherwise, you don't see any attachments at all. And `knowledge` because I plan to add some of the modules depending on this to the document settings wizard